server/pkg/asset/asset.go

```diff
@@ -0,0 +1,70 @@
+package asset
+
+import (
+	"time"
+
+	"github.com/reearth/reearth-cms/server/pkg/project"
+	"github.com/reearth/reearth-cms/server/pkg/user"
+)
+
+type Asset struct {
+	id          ID
+	project   	*project.Project
```

Comment:
```suggestion
```

This is a basic mistake. An aggregate should not have other aggregates. The implementation will quickly become chaotic. Aggregates are indirectly connected to each other only by ID.